optaplanner-docs/src/main/asciidoc/QuickStart/SpringBoot/SpringBootQuickStart-chapter.adoc

```diff
@@ -17,18 +17,17 @@ You will build a REST application that optimizes a school timetable for students
 image::QuickStart/SpringBoot/timeTableAppScreenshot.png[]
 
 Your service will assign `Lesson` instances to `Timeslot` and `Room` instances automatically
-by using AI to adhere to hard and soft scheduling _constraints_, such as:
+by using AI to adhere to hard and soft scheduling _constraints_, such as the following examples:
 
 * A room can have at most one lesson at the same time.
+//consider `one lesson at a time`
 * A teacher can teach at most one lesson at the same time.
 * A student can attend at most one lesson at the same time.
 * A teacher prefers to teach in a single room.
+//Consider: ..to teach all lessons in the same room.
```

Comment:
"all" would imply that if one lesson is out, the constraint equally broken as if 4 lessons are in another room.
Would "every" work here?

Reply:
Yes, every is better.